docs/wiki/PipelinesDesign.md

```diff
@@ -95,10 +95,12 @@ Note that, for the deployments we have to account for certain [prerequisites](#p
 
 #### Removal
 
-The removal phase is strongly coupled with the previous deployment phase. Fundamentally, we want to remove any test-deployed resource after its test concluded. If we would not, we would generate unnecessary costs and may temper with any subsequent test. Some resources may require a dedicated logic to be removed. This logic should be stored alongside the generally utilized removal script in the `.utilities/pipelines/resourceRemoval` folder and be referenced by the `Initialize-DeploymentRemoval.ps1` script that orchestrates the removal.
+The removal phase is strongly coupled with the previous deployment phase. Fundamentally, we want to remove any test-deployed resource after its test concluded. If we would not, we would generate unnecessary costs and may temper with any subsequent test. Some resources may require a dedicated logic to be removed.
```

Comment:
Even if this mainly comes from before I think there's too much information here now that we have a dedicated page for removal. I'd suggest to limit this paragraph to really 2 sentences and point to the dedicated page for details. I'd suggest to avoid expressions like "fundamentally" and "if we wouldn't" but to go straight to the point with shorter and more clear sentences. e.g. something like
```suggestion
The removal phase takes care of removing all resources deployed as part of the previous deployment phase. The reason is twofold: keeping validation subscriptions costs down and allow deployments from scratch at every run.

For additional details on how removal works please refer to the dedicated [Removal action](PipelineRemovalAction) page.
```